### DISCUSSION TITLE
Text size when loading ADFs

### DISCUSSION THREAD

**Discussion by norwichrunner** [BlitterStudio/amiberry#927] [Q&A] [answered]
When loading ADF files to DF0:  from folders containing multiple versions of a game Its sometimes find it difficult to navigate to the correct file.  Is it possible force Amiberry to show the end of the path (instead of the start) or to even make the font smaller in the GUI to fit the file path text in to the box?  Not a huge deal or anything but just wondered.

Thank you!

**Comment by midwan** [comment, marked as answer]
This is already implemented in the upcoming next release :)

**Reply by norwichrunner** [reply]
Oh wow, that was good timing!  Thank you :)
